include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,553 @@
+
+    /* CustomBorderColorFeaturesEXT *//* VK_EXT_custom_border_color */
+    //VkBool32           customBorderColors;
+    //VkBool32           customBorderColorWithoutFormat;
+
+    /* DepthClipControlFeaturesEX *//* VK_EXT_depth_clip_control */
+    //VkBool32           depthClipControl;
+
+    /* DepthClipEnableFeaturesEXT *//* VK_EXT_depth_clip_enable */
+    //VkBool32           depthClipEnable;
+
+    /* DescriptorIndexingFeatures *//* VK_EXT_descriptor_indexing *//* MOVED TO Vulkan 1.2 Core  */
+
+    /* DeviceMemoryReportFeaturesEXT *//* VK_EXT_device_memory_report */
+    //VkBool32           deviceMemoryReport;
+    
+    /* ExtendedDynamicStateFeaturesEXT *//* VK_EXT_extended_dynamic_state */
+    //VkBool32           extendedDynamicState;
+
+    /* ExtendedDynamicState2FeaturesEXT *//* VK_EXT_extended_dynamic_state2 */
+    //VkBool32           extendedDynamicState2;
+    //VkBool32           extendedDynamicState2LogicOp;
+    //VkBool32           extendedDynamicState2PatchControlPoints;
+
+    /* FragmentDensityMapFeaturesEXT *//* VK_EXT_fragment_density_map */
+    //VkBool32           fragmentDensityMap;
+    //VkBool32           fragmentDensityMapDynamic;
+    //VkBool32           fragmentDensityMapNonSubsampledImages;
+
+    /* FragmentDensityMap2FeaturesEXT *//* VK_EXT_fragment_density_map2 */
+    //VkBool32           fragmentDensityMapDeferred;
+
+    /* FragmentShaderInterlockFeaturesEXT *//* VK_EXT_fragment_shader_interlock */
+    bool fragmentShaderSampleInterlock = false;
+    bool fragmentShaderPixelInterlock = false;
+    bool fragmentShaderShadingRateInterlock = false;
+
+    /* GlobalPriorityQueryFeaturesEXT *//* VK_EXT_global_priority */
+    /* GlobalPriorityQueryFeaturesKHR *//* VK_KHR_global_priority */
+    //VkQueueGlobalPriorityKHR    globalPriority;
+
+    /* GraphicsPipelineLibraryFeaturesEXT *//* VK_EXT_graphics_pipeline_library */
+    //VkBool32           graphicsPipelineLibrary;
+
+    /* HostQueryResetFeatures *//* VK_EXT_host_query_reset *//* MOVED TO Vulkan 1.2 Core */
+    
+    /* Image2DViewOf3DFeaturesEXT *//* VK_EXT_image_2d_view_of_3d */
+    //VkBool32           image2DViewOf3D;
+    //VkBool32           sampler2DViewOf3D;
+
+    /* ImageRobustnessFeaturesEXT *//* VK_EXT_image_robustness *//* MOVED TO Vulkan 1.3 Core */
+
+    /* ImageViewMinLodFeaturesEXT *//* VK_EXT_image_view_min_lod */
+    //VkBool32           minLod;
+
+    /* IndexTypeUint8FeaturesEXT *//* VK_EXT_index_type_uint8 */
+    //VkBool32           indexTypeUint8;
+    
+    /* InlineUniformBlockFeaturesEXT *//* VK_EXT_inline_uniform_block *//* MOVED TO Vulkan 1.3 Core */
+
+    /* LineRasterizationFeaturesEXT *//* VK_EXT_line_rasterization */
+    //VkBool32           rectangularLines;
+    //VkBool32           bresenhamLines;
+    //VkBool32           smoothLines;
+    //VkBool32           stippledRectangularLines;
+    //VkBool32           stippledBresenhamLines;
+    //VkBool32           stippledSmoothLines;
```

Comment:
save the comments, but don't expose, cause this feature introduces new/more pipeline state with `VkPipelineRasterizationLineStateCreateInfoEXT`